source/pattern-library/application-framework/login-page/index.md

```diff
@@ -1,12 +1,79 @@
 ---
 title: Login Page
-author: serena
-layout: page
+author: gcardoso
+layout: page-tabs
 ---
-<p class="tn-prototype tn-prototype-inverse"><a title="Login Page Sample" href="https://rawgit.com/patternfly/patternfly/master/tests/login.html" target="_blank"><img class="alignnone wp-image-3625" src="{{site.baseurl}}assets/img/login.png" alt="login" width="1200" height="687" /></a></p>
-<p class="reference-markup"><a class="collapse-toggle collapsed" data-toggle="collapse" aria-expanded="false" aria-controls="login-layout-markup" href="#login-layout-markup">Reference Markup</a></p>
-<div class="collapse" id="login-layout-markup">
-  <pre class="prettyprint">
+<div class="tab-content">
+  <div role="tabpanel" class="tab-pane active" id="overview">
+    <p>The login page allows the user to gain access to an application by entering their username and password. Usage scenarios related to the login page include:</p>
+    <ul>
+      <li>A user navigates to an application and is presented with a login page as a way to gain access to the application. There are two possible results:
+        <ul>
+          <li>Authentication is successful and the user is directed to the application landing page.</li>
+          <li>Authentication fails and the user remains on the login page. If authentication fails, the screen should show an informational or error messages about the failure.</li>
+        </ul>
+      </li>
+      <li>A user is automatically logged out due to inactivity. In this event, they will be returned to the login page, which will display an informational message explaining what happened. Once the user logs in again, they should be taken back to the page they were previously on before being timed out. Thirty minutes is the suggested duration before a session timeout, but this is subject to change based on your product’s security requirements.</li>
+      <li>A user has forgotten their username and/or password. A link is available to begin the process to reset this information. Once the user clicks on one of these links, the contents of the login page is replaced with fields specific to recovering their username and/or password. There are a number of different ways the user could recover their password. This pattern does not dictate which methods an application should follow. Some options include:
+        <ul>
+          <li>The user could provide their e-mail and be sent a temporary password or a link to reset their password.</li>
+          <li>The user could answer a security question.</li>
+          <li>The user could get a message explaining that they have to contact a specific person.</li>
+        </ul>
+      </li>
+    </ul>
+    <p class="tn-prototype tn-prototype-inverse">
+      <a href="https://rawgit.com/patternfly/patternfly/master/tests/login.html" target="_blank">

```

Comment:
Instead of using a link to open a new window, include the login.html test page instead. I've worked this out for the navigation page using a seamless iframe.